app/src/main/kotlin/batect/cli/commands/HelpCommand.kt

```diff
@@ -59,10 +59,7 @@ class HelpCommand(
     private fun nameFor(option: OptionDefinition): String {
         val longNamePart = if (option.acceptsValue) "${option.longOption}=${option.valueFormatForHelp}" else option.longOption
 
-        return when {
-            option.shortName == null -> "    $longNamePart"
-            else -> "${option.shortOption}, $longNamePart"
-        }
+        return option.shortName?.let { "${option.shortOption}, $longNamePart" } ?: "    $longNamePart"
```

Comment:
Sorry, this is far less clear to me. Please revert this change.

Reply:
I've mostly revered the change, but still applied the "'when' with subject should be used" suggestion from IntelliJ. Let's see if you like it.